### PR TITLE
[project:create|update] add language aliases

### PR DESCRIPTION
**Pull request by bhageena**
This is not the best way to cater to the change request and might require a touch-up after #18 is merged. Additionally, this change also needs to be propagated to the plugins repo.

### CODE REVIEW

**Review by rabbah — APPROVED** [review]

**Comment by joshuaauerbachwatson** [comment]
After the merge of PR #18, this PR has conflicts and so would need rebasing.   Also, @bhageena indicated in the initial comment that he would fine-tune this after that merge.   So, I'm holding off on reviewing this until it has been adjusted.

**Review by joshuaauerbachwatson — APPROVED** [review]
Looks good.

**Comment by joshuaauerbachwatson** [comment]
@bhageena I approved this for content but am not able to merge it (easily).  It developed conflicts.   This might be because another PR of my own touches `project.ts` (trivially, I'm surprised if that caused a conflict).

Could you investigate and fix and then I will merge.

**Comment by rabbah** [comment]
@joshuaauerbachwatson you can resolve the conflict using github's ui (click resolve conflicts) - they appear straight forward. Then you can proceed to merge.

**Comment by rabbah** [comment]
I went ahead and did it - please check the last commit.

File: src/generator/project.ts
Comment (rabbah) [inline src/generator/project.ts:149]:
```suggestion
```

**Comment by joshuaauerbachwatson** [comment]
Yup.   Thanks.

**Pull request merged**
